miner/bid_simulator.go

```diff
@@ -758,6 +758,10 @@ func (r *BidRuntime) commitTransaction(chain *core.BlockChain, chainConfig *para
 		env.state.RevertToSnapshot(snap)
 		env.gasPool.SetGas(gp)
 		return err
+	} else if noRevertible && receipt.Status == types.ReceiptStatusFailed {
+		env.state.RevertToSnapshot(snap)
+		env.gasPool.SetGas(gp)
```

Comment:
I think no need to care env now, also L758&L759 can be deleted

Reply:
I think the developer just copy line 758-759, if no need we should delete both